.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
```

Comment:
I think we should use the default ones that come from scikit-hep cookie cutter: https://github.com/scikit-hep/cookie/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/.flake8%5Esetuptools%2Cpybind11

Reply:
This PR is just porting them over from `setup.cfg` so that they still exist. If we want to change them we should do that in another PR.